design-tokens/build.js

```diff
@@ -141,8 +141,8 @@ function buildThemedTokens(themedTokenGroups, baseDesignTokenGroup) {
                 fileHeader: 'licence-header',
                 selector:
                   theme.name === DEFAULT_THEME
-                    ? ':root'
-                    : `html[${THEME_DATA_ATTRIBUTE}='${theme.name}'], [${THEME_DATA_ATTRIBUTE}='${theme.name}']`,
+                    ? `:root, [${THEME_DATA_ATTRIBUTE}='${theme.name}']`
```

Comment:
It seems like there might be an issue with the change, at least in a storybook; changing the theme doesn't seem to work anymore. To double check if we might also have this issue in the product when using this version.

Reply:
Yeah, I broke it. See my comment https://github.com/SonarSource/echoes-react/pull/220#discussion_r1876343837